### PR TITLE
[New][Author] Add react-block-ui w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by TheSharpieOne**
Pull request for issue: #11665

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Availity/react-block-ui.git
 * Official website (optional, not the repository): https://availity.github.io/react-block-ui/
 * NPM package url (optional): https://www.npmjs.com/package/react-block-ui
 * License and its reference: MIT: [Github LICENSE file](https://github.com/Availity/react-block-ui/blob/master/LICENSE) | [package.json license property](https://github.com/Availity/react-block-ui/blob/master/package.json#L55)
 * GitHub popularity (required):
   - Count of watchers: 22
   - Count of stars: 39
   - Count of forks: 6
 * NPM download stats (optional):
   - Downloads in the last day: 343
   - Downloads in the last week: 1,404
   - Downloads in the last month: 5,502

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
   * It is longer than 50 chars, but that is desired to maintain clarity based on previous contributions.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/react-block-ui/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  "license": "MIT",
+  "author": "Evan Sharp <evan.sharp@availity.com> (https://github.com/TheSharpieOne)",
+  "npmName": "react-block-ui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "reactblockui.min.js*",
+        "style.css"
```
Comment (sufuf3) [inline ajax/libs/react-block-ui/package.json]: @TheSharpieOne thank you for your contribution.
May I ask the the `reactblockui.min.js` is compressed from `reactblockui.js`?
If yes. CDNJS can host the `reactblockui.js*` file.

Would you want to add `reactblockui.js`?
If yes.
I think we can modify from A to B as the followings.

- A 
```js
    {
      "basePath": "dist",
      "files": [
        "reactblockui.min.js*",
        "style.css"
      ]
    }
```

- B
```js
    {
      "basePath": "dist",
      "files": [
        "**/+(reactblockui.!(cjs|es)|style)*+(css|js|map)"
      ]
    }
```

What do you think?
Thank you.

Comment (TheSharpieOne) [inline ajax/libs/react-block-ui/package.json]: Yes, I suppose it wouldn't hurt to add the non-minified file. I will update the PR when I can. Thank you.

**Comment by TheSharpieOne** [comment]
Sorry for the delay (had to clone the repo and even with a shallow clone my work proxy was killing it). I made the changes you suggested/requested and amended the commit. I also rebased so my commit is not far from the HEAD of master.

File: ajax/libs/react-block-ui/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "redux",
+    "react",
+    "component",
+    "components",
+    "react-component",
+    "ui"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Availity/react-block-ui.git"
```
Comment (extend1994) [inline ajax/libs/react-block-ui/package.json]: Can we follow the upstream which will be https://github.com/Availity/react-block-ui.git?

File: ajax/libs/react-block-ui/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "license": "MIT",
+  "author": "Evan Sharp <evan.sharp@availity.com> (https://github.com/TheSharpieOne)",
+  "npmName": "react-block-ui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/+(reactblockui.!(cjs|es)|style)*+(css|js|map)"
```
Comment (extend1994) [inline ajax/libs/react-block-ui/package.json]: Let's modify this to `**/+(reactblockui.!(cjs|es)|style)*.+(css|js|map)` which add a `.` between `*` and +(css|js|map). Though the results are the same but the latter one make more sense. Thank you.

**Comment by TheSharpieOne** [comment]
@extend1994 The changes you requests have been made. I amended the original commit and rebased. Should be good to go now.

**Pull request merged**